app/players/elves/elf.py

```diff
@@ -0,0 +1,12 @@
+from app.players.player import Player
+from abc import ABC
```

Comment:
Add a blank line between different import groups and order them correctly to follow PEP-8